### DISCUSSION TITLE
public domain BioArt from NIH/NIAID - stay ahead of the curve

### DISCUSSION THREAD

**Discussion by zivy** [IBEXImagingCommunity/ibex_imaging_knowledge_base#237] [Announcements]
The beta-release of the [NIH BIOART resource](https://bioart.niaid.nih.gov/) is up and running. Free, public domain, scientifically accurate collection of more than 2000 science and medical visuals across multiple categories, including: [Viruses](https://bioart.niaid.nih.gov/discover?bioartCategory=500049), [Anatomy](https://bioart.niaid.nih.gov/discover?bioartCategory=500037), [Cells and organelles](https://bioart.niaid.nih.gov/discover?bioartCategory=500041), [Lab equipment](https://bioart.niaid.nih.gov/discover?bioartCategory=500043), [Proteins](https://bioart.niaid.nih.gov/discover?bioartCategory=500047), [Animals](https://bioart.niaid.nih.gov/discover?bioartCategory=500047), [People](https://bioart.niaid.nih.gov/discover?bioartCategory=500291), ...

<p align="center">
<a href="https://bioart.niaid.nih.gov/"><img width="436" alt="NIH BIOART resource" src="https://github.com/user-attachments/assets/b31f1320-6456-4edb-91b1-42253ad015a6"></a>
</p>

**Comment by radtkea** [comment]
Awesome! Thanks so much for sharing this great resource for the community @zivy!

**Reply by DasEFX130** [reply]
These are really cute! Thank you @zivy
